src/mocks/server.js

```diff
@@ -0,0 +1,6 @@
+/* eslint import/no-extraneous-dependencies:off */
+import { setupServer } from 'msw/node';
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.